plan/config_test.go

```diff
@@ -116,3 +133,156 @@ func TestParseBehavior(t *testing.T) {
 		assert.Equal(t, tt.want, parseBehavior(tt.give))
 	}
 }
+
+func TestParseSkipBehavior(t *testing.T) {
+	tests := []struct {
+		give             string
+		desc             string
+		wantFilters      []Filter
+		wantBehaviorName string
+		wantError        error
+	}{
+		{
+			give: "foo=client:c+server:b",
+			desc: "single filter.",
```

Comment:
nit: 'desc' should usually be the first one in the list because it's the name of the test case